rclrs/Cargo.toml

```diff
@@ -13,16 +13,11 @@ lock_api = "0.4.5"
 cstr_core = "0.2"
 cty = "0.2"
```

Comment:
Aren't `cstr_core` and `cty` also no_std-only dependencies?

Reply:
Removed